packages/cascara/src/specs/Table/Table.spec.mdx

```diff
@@ -1,7 +1,155 @@
 ---
 title: Table
+status: Draft
+
 ---
 
 # Table
 
-Let's see if this shows up at our application root.
+## Table Types
+
+- Data table The standard table layout with a table header and rows of data
+- Grid view - more of a gallery view. No table header. Could be considered a list. Need to discuss if we want to make this a separate component or a variation on a table.
+
+## Table Locations
+
+### Admin
+
+- BRC (Grid)
+- Storefront Designer (grid)
+- Bundles (grid)
+- Role Bundles (grid)
+- Onboarding Activities/Schedule
+- Onboarding Activities/Performance
+- Service Department
+- Service Teams
+- Barista FAQS
+- Barista FAQS/Topics
+- Users
+- Integrations/CSV/Import History
+- Integrations/CSV/Import Schedule
+- Integrations/Email/Barista Inboxes
+- Reports
+- Locations
+
+### Barista
+
+- None
+
+### Caffeine
+
+- Entities
+- Training
+- Variables
+- Logs (future)
+- Users (future)
+- Configurations (future)
+- Localization(future)
+
+### Status
+
+- Packages
+
+### Doppio
+
+- Applications
+- Applications/[Application]
+- Intents
+- Intents/[Intent]
+- FAQs
+- FAQS/[FAQ]
+- Entities
+- Archetype Conditions
+- Attributes
+- Hooks
+- Synonyms
+- Synonyms/[Synonym]
+- Variables
+- Variables/[Variable]
+- Training
+- Spell Corrector Dictionary
+- Utilities
+- ESP Service Departments
+- Service Team Classifications
+- Task Classifications
+- Service Departments
+- Service Teams
+- Task Categories
+- Testing Reports
+- Test Phrases
+- Test Sets
+- Tasks
+- Tasks/[Task]
+- Categories
+- Categories/[Category]
+- Outages
+- Outages/[Outage]
+- Integrations
+- Integrations/[Integration]
+- Service Requests
+- Service Request/[Service Request]
+- Model Bindings
+- Widgets
+- Users
+- Users/[User]
+- Locations
+- Assets
+- Job Roles
+- Classifiers
+- Data Sources
+- Data Sources/[Data Source]
+- Annotations
+- Annotations/[Annotation]
+- Smart Rules
+- Smart Rules/[Rule]
+- Flagged Tickets
+- Surveys
+- Configurations
+- Attachments
+- Business Events
+- Localization
+
+## Existing Table Functionality
+
+### CRUD Capabilities
+
+- Ability to add a data row
+- Ability to edit a table rown (inline optional)
+- Ability to delete a row (optional)
+
+### Table Header
+
+- Column headers with labels and optional descriptions
+- Sortable column headers
+
+### Pagination
```

Comment:
This feature already exists as [Pagination](https://github.com/Espressive/cascara/tree/develop/packages/cascara/src/ui/Pagination) component, but it's good to have it here to mark it as `out of scope`.

Reply:
Yeah... Maybe we need a related components section. Could we bake in a requirement though to include the pagination component into any implementation of the table component?

Reply:
- [ ] Yes. Let's add a "Related Components" section for now while we figure this out. This will allow us to define separate area needs in the spec without having to flesh them out.